src/app/models/profile.ts

```diff
@@ -6,8 +6,11 @@ export class Profile {
     bio?: string;
     username?: string;
     url?: string;
-    emails?: Email[];
-    primaryEmail?: Email;
+    emails?: string[];
```

Comment:
The way we could model this is with some sort of map and labels, but tbh I want to understand what you are acutally worried about ;-)